### PR TITLE
Initial modules - Sunpath

### PR DESCRIPTION
**Pull request by devngc**
Initial modules are added that can be used to create a Sunpath. The [unit test](https://github.com/devngc/ladybug-vtk/blob/sunpath/tests/sunpath_test.py#L12-L67) shows a sample.

### CODE REVIEW

File: ladybug_vtk/model.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+"""A VTK representation of HBModel."""
```
Comment (mostaphaRoudsari) [inline ladybug_vtk/model.py:1]: We should probably keep this module in honeybee-vtk. In the ladybug world, there is no such concept as a model.

File: ladybug_vtk/to_vtk.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+"""Functions to create VTK objects from Ladybug Objects."""
+
+import vtk
+from typing import List
+from ladybug_geometry.geometry3d import Point3D, Polyline3D
+from .polydata import PolyData
+
+
+def create_points(points: List[Point3D]) -> PolyData:
```
Comment (mostaphaRoudsari) [inline ladybug_vtk/to_vtk.py:9]: We should probably name the functions similar to other translation libraries: https://github.com/ladybug-tools/ladybug-rhino/blob/master/ladybug_rhino/togeometry.py

```suggestion
def to_vtk_points(points: List[Point3D]) -> PolyData:
```

File: ladybug_vtk/to_vtk.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    """Create Polydata from a Polyline."""
```
Comment (mostaphaRoudsari) [inline ladybug_vtk/to_vtk.py:10]: This is a bit confusing. Shouldn't this method only return a PolyData with points? Or is this the one to create a polyline?

File: requirements.txt [hunk 4]
```diff
@@ -1 +1,2 @@
-ladybug-geometry==1.23.19
\ No newline at end of file
+lbt-ladybug==0.25.96
```
Comment (mostaphaRoudsari) [inline requirements.txt:1]: Shouldn't you only need the geometry library? Why do we need `lbt-ladybug` here?

File: tests/sunpath_test.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+from pathlib import Path
+from ladybug.epw import EPW
+from ladybug.location import Location
+from ladybug.sunpath import Sunpath, Point3D, Vector3D
+from ladybug_vtk.to_vtk import create_polyline, create_points
+from ladybug_vtk.model_dataset import ModelDataSet
+from ladybug_vtk.model import Model
+
+
+def test_sunpath(temp_folder):
```
Comment (mostaphaRoudsari) [inline tests/sunpath_test.py:12]: This is a great example. We can write this as a function and inject it to the ladybug Sunpath class. I can show you some sample code.

File: tests/sunpath_test.py [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+    polylines = sp.hourly_analemma_polyline3d(
+        origin=origin, daytime_only=True, radius=radius)
+    sp_polydata = [create_polyline(pl) for pl in polylines]
+    sp_dataset = ModelDataSet(name='Polylines', data=sp_polydata)
+
+    # monthly arcs
+    arcs = sp.monthly_day_arc3d()
+    arc_polylines = [arc.to_polyline(divisions=10)for arc in arcs]
+    arc_polydata = [create_polyline(arc) for arc in arc_polylines]
+    arc_dataset = ModelDataSet(name='Arcs', data=arc_polydata)
```
Comment (mostaphaRoudsari) [inline tests/sunpath_test.py:34]: For now, it is fine to use a Polyline but hopefully, we can support arcs at some point.

File: requirements.txt (hunk 4 above)
Comment (mostaphaRoudsari) [reply requirements.txt:1]: I see that for now, you are using it for tests. In that case, you probably should move this dependency to dev-requirements which get installed for running tests.

File: ladybug_vtk/model.py (hunk 1 above)
Comment (mostaphaRoudsari) [reply ladybug_vtk/model.py:1]: I saw how you are using it in your test example. We should probably update the description.

File: tests/sunpath_test.py (hunk 5 above)
Comment (mostaphaRoudsari) [reply tests/sunpath_test.py:12]: Here is the module that extends honeybee and adds radiance properties to it:
https://github.com/ladybug-tools/honeybee-radiance/blob/master/honeybee_radiance/_extend_honeybee.py

The module is then called from inside `__init__.py`. We can work on that after merging this PR.

**Review by mostaphaRoudsari — APPROVED** [review]
Hi @devngc, I left a few comments - mostly suggestions and one for the next step to inject the function that you wrote as a method of the `Sunpath` class.

File: tests/sunpath_test.py (hunk 5 above)
Comment (mostaphaRoudsari) [reply tests/sunpath_test.py:12]: I sent a PR to address this comment: https://github.com/devngc/ladybug-vtk/pull/2

File: requirements.txt (hunk 4 above)
Comment (mostaphaRoudsari) [reply requirements.txt:1]: I was wrong. We actually need the core library to be able to extend the classes.

File: tests/sunpath_test.py (hunk 6 above)
Comment (devngc) [reply tests/sunpath_test.py]: https://github.com/ladybug-tools/ladybug-vtk/issues/3

**Pull request merged**